h/static/scripts/bridge.coffee

```diff
@@ -118,7 +118,22 @@ class Bridge
       options = $.extend {}, options, {origin: '*'}
     channel = Channel.build(options)
 
+class BridgeFactory
+
+  constructor: ->
+
+  _createBridge: ->
+    # Set up the bridge plugin, which bridges the main annotation methods
+    # between the host page and the panel widget.
+    options =
+      scope: 'annotator:bridge'
+    @_bridge = new Bridge(options)
+
+   $get: ->
+     @_createBridge() unless @_bridge?
+     @_bridge
+
 if angular?
-  angular.module('h').value('Bridge', Bridge)
+  angular.module('h').provider('bridge', BridgeFactory)
```

Comment:
Unless you're thinking that the options should be exposed, in which case you want the provider to have the options on as a member variable.